.github/ISSUE_TEMPLATE/crash.yml

```diff
@@ -0,0 +1,54 @@
+name: Crash report
+description: A hard crash of the interpreter, possibly with a core dump
+labels: ["type-crash"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          This form is for hard crashes of the Python interpreter, segmentation faults, failed C-level assertions, and similar.
+          Exceptions unexpectedly raised from stdlib Python functions count as bugs rather than crashes.
+
+          The CPython interpreter is written in a different programming language, C.
+          A "CPython crash" is when Python itself fails, leading to a traceback in the C stack.
```

Comment:
Perhaps include that generally "Segmentation fault" or "Aborted (core dumped)" or etc are printed to console in these cases, and if you don't see such markers it probably isn't an interpreter crash?

Reply:
Is that true? I think you can get some silent segfaults where the interpreter just instantly exits. (Not an expert on these matters, though!)

Reply:
I also don't know -- the intent was to clarify "what is a crash", but it isn't that important!